app/forms/prior_authority/steps/case_detail/defendant_form.rb

```diff
@@ -0,0 +1,24 @@
+module PriorAuthority
+  module Steps
+    module CaseDetail
+      class DefendantForm < ::Steps::BaseFormObject
+        attribute :maat, :string
+        validates :maat, presence: true
+
+        private
+
+        def persist!
+          existing = application.defendant
```

Comment:
You could make this a bit terser with `application.defendant || application.build_defendant`, then you don't need to use `&.` or an `if` statement